Managers/LuaMan.cpp

```diff
@@ -403,6 +405,38 @@ namespace RTE {
 		return isDefined;
 	}
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	const std::vector<std::string> & LuaMan::DirectoryList(const std::string &filePath) {
+		std::string fullPath = System::GetWorkingDirectory() + filePath;
+
+		m_Paths.clear();
+
+		for (const auto &directoryEntry : std::filesystem::directory_iterator{fullPath}) {
+			if (directoryEntry.is_directory()) {
+				m_Paths.push_back(directoryEntry.path().filename().string());
```

Comment:
Use `generic_string` to get a `nicely/separated/path/` instead of `stupid\\microsoft\\bullshit`.